.github/workflows/black.yml

```
@@ -6,17 +6,32 @@ jobs:
   lint:
     runs-on: ubuntu-20.04
     steps:
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
 
       - name: Set up Python 3.9
-        uses: actions/setup-python@v2
+        uses: actions/setup-python@v4
         with:
-          python-version: 3.9
-
+          python-version: '3.9'
       - name: Install dependencies
         run: |
           pip install black==22.3.0
```

I don't know why we are using black version `22.3.0`. I think we could be using always the latest one 😄 

```suggestion
          pip install black
```

---

I am going to set this to `24.10.0` as it is the version set in insight-plugin.